.github/workflows/upload-sourcemaps.yml

```diff
@@ -34,6 +34,7 @@ jobs:
           cp -r dist/mdot dist/app
           cp -r dist/mdot dist/share
           cp -r dist/mdot dist/p
-          declare -x VERSION=mdot@$(sentry-cli releases propose-version)
-          sentry-cli releases set-commits $VERSION --auto --ignore-missing
-          sentry-cli releases files $VERSION upload-sourcemaps --validate --log-level info --strip-common-prefix ./dist/ --ignore mdot
+          VERSION="mdot@$(sentry-cli releases propose-version)"
+          export VERSION
+          sentry-cli releases set-commits "$VERSION" --auto --ignore-missing
```

Comment:
The linter pointed out that $VERSION should have quotes around it (same for the next line)